libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -333,8 +350,20 @@ bool AP_Arming::battery_checks(bool report)
             }
             return false;
         }
-    }
 
+        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
```

Comment:
Mostly because I didn't think of it. Although I admit to having mixed
feelings about any loop that iterates over a list of parameters as it's
easy to raise the constant without adding a parameter.

I can add a loop and push it today.
On Mar 2, 2016 12:27 PM, "Tom Pittenger" <notifications@github.com> wrote:

> In libraries/AP_Arming/AP_Arming.cpp
> <https://github.com/diydrones/ardupilot/pull/3604#discussion_r54776432>:
>
> >
> > +        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
> > +            if (report) {
> > +                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
>
> why not put this into a loop?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3604/files#r54776432>.
>